### DISCUSSION TITLE
Cant add project in Community exchange for collaboration

### DISCUSSION THREAD

**Discussion by Irrfaansaul** [community/community#56915] [GitHub Education]
### Hi everyone,

I have added all the details to make it opensource for contribution but I cant it has issues with issues I have made issues template but still its the same,

repo link - https://github.com/jarvismsk/MindSOS

Repository description
README.md
License

Issues
You must have Issues ready before you submit the repository.

CONTRIBUTING.md

**Comment by samus-aran** [comment]
Hi @jarvismsk 

It appears you have resolved your issue as I can see the project on the Community Exchange. Well done on creating such a mindful OSS project 🌈 👏 ⭐ 

I'll be closing this discussion due to it being resolved.

**Comment by alecxcode** [comment]
Hello, please reopen the issue. The same problem:
Repository: https://github.com/alecxcode/edm
Cannot resolve the problem and submit it. The exchange portal asks a vague question: You must have Issues ready before you submit the repository. What does that mean, what are not ready. All other repositories have empty issues and no templates.
